dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/config/SpringConfigSecurityWebapp.java

```diff
@@ -72,7 +67,10 @@ protected void configure(HttpSecurity http) throws Exception {
         .permitAll()
         .and()
         .rememberMe()
-        .tokenRepository(inMemoryTokenRepositoryImpl)
+        .rememberMeParameter("remember-me")
+        .key("ajksh34h534h34v5gjh")
```

Comment:
Sollte das nicht aus der `application.yml` kommen?

Reply:
ja, hatte ich schon in filterchain config, habs vergessen in securityadapter config nachzuziehen. done. danke